misc-tools/dj_make_chroot.in

```diff
@@ -149,6 +149,14 @@ fi
 [ -z "$CHROOTDIR" ] && error "No chroot directory given or default known."
 [ -z "$ARCH" ]      && error "No architecture given or detected."
 
+ARCHLIST=$DEBIAN_ARCHLIST','
+if [ "$DISTRO" = "Ubuntu" ]; then
+    ARCHLIST=$UBUNTU_ARCHLIST','
+fi
```

Comment:
This is a nice trick, but I'd also prefix the `ARCHLIST` with a comma and then match on `,$ARCH,` since otherwise `$ARCH` could match a suffix of an existing architecture.